Utils/github_workflow_scripts/check_if_partner_approved_label_exists.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+import sys
+from github import Github
+from blessings import Terminal
+import argparse
+import urllib3
+from github.Repository import Repository
+from github.PullRequest import PullRequest
+from demisto_sdk.commands.common.tools import get_pack_metadata, get_pack_name
+from Utils.github_workflow_scripts.utils import timestamped_print, get_support_level
+
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+print = timestamped_print
+
+PARTNER_APPROVED_LABEL = 'Partner-Approved'
+
+
+def arguments_handler():
+    """ Validates and parses script arguments.
+
+     Returns:
+        Namespace: Parsed arguments object.
+    """
+    parser = argparse.ArgumentParser(description='Check if Partner-Approved label exists.')
+    parser.add_argument('-p', '--pr_number', help='The PR number to check if the label exists.')
+    parser.add_argument('-g', '--github_token', help='The GitHub token to authenticate the GitHub client.')
+    return parser.parse_args()
+
+
+def get_pack_support_level(file_paths: list[str]) -> str:
+    """
+    :param file_paths: the paths of files that are being changed in the PR
+    :return: pack support level
+    """
+    pack_dirs_to_check_support_levels_labels = set()
+
+    for file_path in file_paths:
+        try:
+            if 'Packs' in file_path and (pack_name := get_pack_name(file_path)):
+                pack_dirs_to_check_support_levels_labels.add(f'Packs/{pack_name}')
+        except Exception as err:
+            print(f'Could not retrieve pack name from file {file_path}, {err=}')
+
+    print(f'{pack_dirs_to_check_support_levels_labels=}')
+    packs_support_levels = get_support_level(pack_dirs_to_check_support_levels_labels)
+    return packs_support_levels
+
+
+def main():
+    """
+    this script is checking that "partner-approved" label exists for a PR
+    in case the label exists the workflow will pass, if the label is missing the workflow will fail
```

Comment:
```suggestion
    This script is checking that "partner-approved" label exists for a PR
    in case the label exists the workflow will pass, if the label is missing the workflow will fail.
```